src/main/java/org/example/app/entity/Twit.java

```diff
@@ -0,0 +1,37 @@
+package org.example.app.entity;
+
+import lombok.Data;
+import org.apache.solr.client.solrj.beans.Field;
+
+import javax.persistence.*;
```

Comment:
There's a good practice to avoid wildcard imports.

Reply:
fixed